### DISCUSSION TITLE
Rename arguments in Hybrid Command

### DISCUSSION THREAD

**Discussion by p-krystian** [Rapptz/discord.py#8365] [Q&A] [answered]
Is a way to set a different argument name showing on Discord instead of name in the code?

**Comment by Soheab** [comment, marked as answer]
You can use the [@app_commands.describe](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.describe) decorator for the description that will show up on discord for the arguments. 
```py
@app_commands.describe(message="The message")
async def say(ctx, message: str):
    ...
```
And for ext.commands (in the help command), you can use the `description=` kwarg in [commands.Parameter](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.parameter).
```py
async def say(ctx, message: str = commands.Parameter(description="The message")):
    ...
```

Both can be used at the same time.

**Reply by p-krystian** [reply]
I found a solution to my problem.

```python
@commands.hybrid_command(name='example', description='Command description')
@app_commands.rename(arg='argument')
async def _example(ctx, arg:str):
    pass
```
[Link to documentation](https://discordpy.readthedocs.io/en/stable/interactions/api.html?highlight=describe#discord.app_commands.rename)

I wanted to edit the name of the argument, not the description.
Thanks for the help!
